### PR TITLE
Run CI on 1.11

### PR DESCRIPTION
**Pull request by Sbozzolo**
Julia 1.11 was just released

### CODE REVIEW

File: .github/workflows/Documentation.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@latest
         with:
-          version: 1
-      - uses: julia-actions/cache@v1
+          version: '1.10'
```
Comment (juliasloan25) [inline .github/workflows/Documentation.yml:21]: why is this using 1.10 instead of 1.11?

**Review by juliasloan25 — APPROVED** [review]

File: .github/workflows/Documentation.yml (hunk 1 above)
Comment (Sbozzolo) [reply .github/workflows/Documentation.yml]: Changed to 1.11

File: .github/workflows/Invalidations.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
     - uses: julia-actions/setup-julia@v1
       with:
-        version: '1'
+        version: '1.10'
```
Comment (juliasloan25) [inline .github/workflows/Invalidations.yml:21]: same for invalidations here

Comment (Sbozzolo) [reply .github/workflows/Invalidations.yml:21]: Invalidations are broken in 1.11 at the moment

**Pull request merged**